.github/workflows/sqa-tools-builder.yaml

```
@@ -0,0 +1,148 @@
+name: Build SQA Tools
+
+on:
+  push:
+# TODO add these lines back       branches:
+#            - main
+#            - "release_*"
+  pull_request:
+      branches:
+          - main
+          - "release_*"
+
+concurrency:
+    group: sqa-tools-builder-${{ github.ref }}
+    cancel-in-progress: true
+
+env:
+  PW_ENVIRONMENT_ROOT: ${{ github.workspace }}/.environment
+
+jobs:
+  package-provisioning-and-ota-scripts:
```

To avoid needing to change the name if we need to add more stuff.
```suggestion
  package-scripts:
```